src/components/NewsletterForm.astro

```diff
@@ -24,8 +24,6 @@
   and hit "Yes, subscribe me to this list"
 </p>
 
-<script is:inline src="https://cdn.outseta.com/emailsignup.min.js"></script>
```

Comment:
note (performance): FYI, As in line script loading blocks the main thread render, I moved this to the Layout.astro with async so that it's loaded in parallel (and occupied in main thread as it's fully downloaded as it is required for waitlist form).